### PR TITLE
chore(editorconfig): added .editorconfig

### PR DESCRIPTION
**Pull request by mastilver**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
Comment (ericelliott) [inline .editorconfig]: This should be 2

Comment (mastilver) [inline .editorconfig]: oups, I made some test and forgot to change it back

Comment (mastilver) [inline .editorconfig]: updated!

**Comment by ericelliott** [comment]
Nice. =)

**Pull request merged**